### ISSUE TITLE
<Progress isIndeterminate /> is not working 

### ISSUE THREAD

**Issue by mcha-dev** [chakra-ui/chakra-ui#4043]
# 🐛 Bug report
`<Progress isInderterminate/>` is not working (even on the documentation page https://chakra-ui.com/docs/feedback/progress#animated-progress)

## 💥 Steps to reproduce

1. `<Progress isIndeterminate /> `
2. nothing is shown 

## 💻 Link to reproduction

Documentation link https://chakra-ui.com/docs/feedback/progress#animated-progress

## 🧐 Expected behavior

Should show progress flashing 

## 🌍 System information

| Software         | Version(s) |
| ---------------- | ---------- |
| Chakra UI        |       ^1.6.2     |
| Browser          |     Firefox       |
| Operating System |     MacOS       |

**Comment by bring-shrubbery** [comment]
I have the same problem 😢

**Comment by mcha-dev** [comment]
![chk](https://user-images.githubusercontent.com/13101565/118976544-53b15000-b99f-11eb-9c5e-aa82fbafd6f7.png)

Found the 🐛  `linear-gradient` is being wrapped in a `url`. Removing the url fixes it.

```css
linear-gradient(
  to right,
  transparent 0%,
  #98529d 50%,
  transparent 100%
)
```

**Comment by mcha-dev** [comment]
As a workaround, passing `transition` prop solves it

```jsx
<Progress 
    isIndeterminate
    transition="linear-gradient(to right, transparent 0%, #98529d 50%, transparent 100%)" 
/>
```

**Comment by bring-shrubbery** [comment]
@mcha-dev The linear gradient is set on the `background-image` right? How would setting it on transition would help? Asking because the workaround you suggested is not working for me.

**Comment by mcha-dev** [comment]
Maybe it's the `background` props? This is what I have now, and it's working (v.1.6.2)
```jsx
import { Progress, ProgressProps } from '@chakra-ui/progress';

export function PageProgress(props: ProgressProps) {
  return (
    <Progress
      h="5px"
      isIndeterminate
      position="absolute"
      top="0"
      right="0"
      left="0"
      zIndex="20"
      bgColor="transparent" // 👈 👇
      transition="linear-gradient(to right, transparent 0%, #98529d 50%, transparent 100%)"
    />
  );
}


```

**Comment by bring-shrubbery** [comment]
The linear gradient is set on the `background-image` property of the child element of the progress wrapper (parent element), so I'm not sure why it's working for you with the `transition` property. I pretty much everything you have except the height setting, and the linear gradient is visible if I set it manually, but not through `transition`

**Comment by mcha-dev** [comment]
You're right, it's not working. Browser was caching hard-coded values on my side :\

**Comment by mcha-dev** [comment]
I think so too. In the mean time, if you're importing a css file somewhere you can try this
```jsx
<Progress className="page-progress-temp" />
```
```css
.page-progress-temp * {
  background-image: linear-gradient(to right, transparent 0%, #98529d 50%, transparent 100%) !important;
}
```

**Comment by mcha-dev** [comment]
I think I found the cause. in packages/styled-system/src/utils/transform-functions.ts
```js
  return isString(value) && value.startsWith("url") ? `url(${value})` : value
``` 
@segunadebayo can you please 👀

**Comment by prescottbreeden** [comment]
Are these changes published yet? Still happening for me on 1.6.2

**Comment by mcha-dev** [comment]
It was tagged for release, just got publish with 1.6.3🎉
